mucommander-core/src/main/java/com/mucommander/ui/main/MainFrame.java

```diff
@@ -206,17 +209,34 @@ public Insets getInsets() {
         };
 
         // Remove any default border the split pane has
-        splitPane.setBorder(null);
+        foldersSplitPane.setBorder(null);
 
         // Add buttons that allow to collapse and expand the split pane in both directions
-        splitPane.setOneTouchExpandable(true);
+        foldersSplitPane.setOneTouchExpandable(true);
 
         // Disable all the JSplitPane accessibility shortcuts that are registered by default, as some of them
         // conflict with default mucommander action shortcuts (e.g. F6 and F8) 
-        splitPane.disableAccessibilityShortcuts();
+        foldersSplitPane.disableAccessibilityShortcuts();
 
+        verticalSplitPane = new JSplitPane( JSplitPane.VERTICAL_SPLIT, true) {
+            @Override
+            public Insets getInsets() {
+                return new Insets(0, 0, 0, 0);
+            }
+        };
+        verticalSplitPane.setBorder(null);
+        verticalSplitPane.setOneTouchExpandable(true);
+        // TODO?
```

Comment:
I see the comment above:
```
// Disable all the JSplitPane accessibility shortcuts that are registered by default, as some of them
// conflict with default mucommander action shortcuts (e.g. F6 and F8)
```
which seems to hold to the verticalSplitPane as well

Reply:
done